architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -34,6 +37,44 @@ you are responsible for updating it to reflect requirements of newer versions of
 the {product-title} master.
 ====
 
+=== General Admission Rules
+Starting in 1.3, OpenShift uses a single admission chain for Kubernetes and OpenShift
+resources.  This changed from 1.2 and before where we had separate admission chains.
```

Comment:
This changed from
ifdef::openshift-enterprise[]
3.2
endif::[]
ifdef::openshift-origin[]
1.2
endif::[]
and earlier, when there were separate admission chains.